src/main/java/taxi/controller/filter/AuthenticationFilterController.java

```diff
@@ -0,0 +1,39 @@
+package taxi.controller.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilterController implements Filter {
```

Comment:
i recommend you to remove "Controller" from name of this class and move it to separate package web.filter

Reply:
Ok